torch/_higher_order_ops/auto_functionalize.py

```diff
@@ -231,6 +232,11 @@ def do_auto_functionalize(
             normalized_kwargs[arg.name] = arg.default_value
 
     unwrapped_kwargs = ctx.unwrap_tensors(normalized_kwargs)  # type: ignore[arg-type]
+    if "self" in unwrapped_kwargs:
```

Comment:
can you also warn for `self_` ?

Reply:
fixed in https://github.com/pytorch/pytorch/pull/130179/commits/b2be36f954b4dfca570008e84437c424c32901da